packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -141,7 +141,9 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
 
   const verifyCode = useVerifyCode({
     onSuccess: () => {
-      verifyEmail.setVerifiedEmail(bookerForm.formEmail);
+      if (!bookerForm.formEmail) return;
+
+      verifyEmail.setVerifiedEmail(bookerForm?.formEmail ?? null);
```

Comment:
If we're doing early return above then this should not be neither null nor undefined no? @Udit-takkar 
```suggestion
      verifyEmail.setVerifiedEmail(bookerForm.formEmail);
```